crates/sgx_validation/src/config.rs

```diff
@@ -0,0 +1,130 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use cryptography::digest::Digest;
+use serde::{Deserialize, Deserializer, Serialize};
+use sgx::parameters::Features;
+
+#[derive(Clone, Deserialize, Debug, Serialize)]
+pub enum SgxFeatures {
+    CET,
+    Debug,
+    EIntKey,
+    KSS,
+    ProvisioningKey,
+}
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct ConfigIntermediate {
```

Comment:
There should not be an intermediate step, you either parse a valid `Config` from TOML or fail, there's no other option

Reply:
That's what happens, it works successfully, or fails.

Reply:
`ConfigIntermediate` does not fit in this process

Reply:
Fixed